### ISSUE TITLE
Track Fault Detection using DL

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#224]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Railway Track fault Detection</br>
:red_circle: **Aim** : Create a model which will classify Defective and Non-Defective Railway Track</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/gpiosenka/railway-track-fault-detection-resized-224-x-224</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by rafiya2003** [comment]
I would like to work on this, frankly I don't know much but I will learn and do. I am a newbie(first year UG student). I want to learn and grow. please give me the opportunity to learn and experience

**Comment by abhisheks008** [comment]
Please do not to spam in every single issue. This might cause escalation as you are breaching Code of Conduct and Contribution Guidelines.

@rafiya2003

**Comment by PiyushBL45t** [comment]
Hi @abhisheks008 I want to contribute and solve this issue.
Name: Piyush Bhujbal
GitHub profile link: https://www.github.com/piyushbl45t
Participant Role: Participant at Social Summer of Code
Approach for this project:
1. I shall first understand the main intention behind this project.
2. Then make things simple by performing data and image preprocessing.
3. Create a Deep Neural Network architecture for the problem statement.
4. Test various neural network types for the problem statement.
5. Prepare a fully fledged report on the various parameters considered while testing the algorithms like
    - Training time.
    - Accuracy.
    - Evaluation Metrics.

**Comment by Sanjay71013** [comment]
Full name : Sanjay Mythili
GitHub Profile Link : https://github.com/Sanjay71013
Email ID : sanjaymythili2002@gmail.com
Participant ID (if applicable): Not applicable
Approach for this Project : I'm planning to solve this classification problem using transfer learning and use Resnet50 and VGG16. I'm also thinking of using Vision Transformer and submit the model which has more accuracy.
What is your participant role? (Mention the Open Source program)
Contributor at Social Summer of Code (SSoC'23)

**Comment by abhisheks008** [comment]
As per the code of conduct and contribution guidelines, issue will be assigned on a first come first serve basis.

Issue assigned to @PiyushBL45t

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link : https://github.com/the-silent-geek
Email ID : udaykir3911@gmail.com
Approach for this Project : This can be implemented by using pre-trained models like vgg16, ResNet, DenseNet, InceptionV3 and MobileNet
What is your participant role? social winter of code 23

**Comment by abhisheks008** [comment]
Issue assigned to you @the-silent-geek
